_std/macros/atom_properties.dm

```diff
@@ -213,6 +213,7 @@ To remove:
 #define PROP_MOB_EXPLOPROT(x) x("exploprot", APPLY_ATOM_PROPERTY_SUM, REMOVE_ATOM_PROPERTY_SUM)
 #define PROP_MOB_DISARM_RESIST(x) x("disarm_resist", APPLY_ATOM_PROPERTY_SUM, REMOVE_ATOM_PROPERTY_SUM)
 #define PROP_MOB_REFLECTPROT(x) x("reflection", APPLY_ATOM_PROPERTY_SIMPLE, REMOVE_ATOM_PROPERTY_SIMPLE)
+#define PROP_MOB_ACIDRES(x) x("acidres", APPLY_ATOM_PROPERTY_SUM, REMOVE_ATOM_PROPERTY_SUM)
```

Comment:
unused

Reply:
Would argue this should exists for completeness' sake, but not entirely opposed to removing it.